tests/Silverback.Integration.Kafka.TestConsumer/Program.cs

```diff
@@ -45,7 +54,8 @@ private static void Connect()
                     BootstrapServers = "PLAINTEXT://localhost:9092",
                     GroupId = "silverback-consumer",
                     AutoOffsetReset = Confluent.Kafka.AutoOffsetReset.Earliest
-                }
+                },
+                Serializer = new JsonMessageSerializer<TestMessage>()
```

Comment:
Shouldn't be necessary.

Reply:
It is necessary, because the FullName of the type is used and not only the Namespace.